File: guanabara-java/src/com/danillo/algoritmos/lista03/Exercicio32.java
package com.danillo.algoritmos.lista03;

import java.util.Scanner;

public class Exercicio32 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        int numero = 4;

        System.out.println("Tente adivinhar o número mágico de 1 a 5: ");
        int chute = entrada.nextInt();

        if (chute == numero){
            System.out.println("Acertou!");
        } else {
            System.out.println("Errou!");
        }
    }
}

File: guanabara-python/src/lista03/exercicio32.py
import random

numero_magico = random.randint(1, 5)
x = int(input("Descubra o número mágico (entre 1 e 5): "))

if x == numero_magico:
    print(f"Acertou! Número mágico: {numero_magico}")
else:
    print(f"Errou! Número mágico: {numero_magico}")
